index.js

```diff
@@ -404,15 +404,19 @@ export default function(value, nominatim_object, optional_conf_parm) {
      */
     function getRegexKeyForKeyFromOsmDefaults(key) {
         var regex_key;
+        var exact_match = false;
 
-        for (var osm_key in osm_tag_defaults) {
+        Object.keys(osm_tag_defaults).forEach(function (osm_key) {
+            if (exact_match === true) {
+                return;
+            }
             if (key === osm_key) { // Exact match.
                 regex_key = osm_key;
-                break;
```

Comment:
> BTW I'm not very proud of the tricky point commit but as we know, will be a greater project to rewrite the lib in es6 for example and transpile it, this way relying on either babel/traceur (not sure to include buble as it don't (intentionnaly) transpile too deep maybe)

Ah, I knew I missed something :smile: Sure, the transpile option would be preferred. You can do this when you want as discussed above. If not, maybe add an comment to explain why this was needed, otherwise, it might get refactored since all tests are passing even with the old code. You might want to add an integration test to ensure that compatibility stays and does not get broken again.

> So of course @simon04 @ypid I can squash commits if we intend to merge without changes.

Yes, I guess you could squash the last commit.